view/adminhtml/web/template/view/certificates-grid.html

```diff
@@ -0,0 +1,54 @@
+<!-- ko if: shouldShowWarning -->
+<p class="message">
+    Please note that you can only view certificates for a customer from the website/store that customer
+    is associated to. This means if you have 2 different Avalara credentials for 2 different stores and one customer
+    uploads certificates to both, only certificates uploaded through the store that customer is associated to will
+    be visible here.
+</p>
+<!-- /ko -->
+<div class="admin__data-grid-wrap admin__data-grid-wrap-static">
+    <table class="data-grid">
+        <thead>
+            <tr>
+                <th class="data-grid-th">State</th>
```

Comment:
Add translation to these column headers and the rest of the strings in this file.